src/Simplify.elm

```diff
@@ -2099,13 +2429,15 @@ negateNegateCompositionErrorMessage =
 
 
 negateCompositionCheck : CompositionCheckInfo -> List (Error {})
-negateCompositionCheck { lookupTable, fromLeftToRight, parentRange, left, right, leftRange, rightRange } =
+negateCompositionCheck { lookupTable, fromLeftToRight, parentRange, left, right, leftRange, rightRange, importLookup } =
     case Maybe.map2 Tuple.pair (getNegateFunction lookupTable left) (getNegateFunction lookupTable right) of
         Just _ ->
             [ Rule.errorWithFix
                 negateNegateCompositionErrorMessage
                 parentRange
-                [ Fix.replaceRangeBy parentRange "identity" ]
+                [ Fix.replaceRangeBy parentRange
+                    (qualifiedToString (qualify ( [ "Basics" ], "identity" ) importLookup))
```

Comment:
Is this ever not going to give us an unqualified `"identity"`? :thinking: 
I think for all the variables and types exposed through the Elm prelude, there is no need to try and do an import lookup, right?

Reply:
There might be a `module`-local shadowed function called `identity`

Reply:
> There might be a shadowed function called identity (as you have suggested; yet to be implemented)

Ugh. You're right... Shucks.